requirements.txt

```diff
@@ -1,3 +1,4 @@
 six
 pytest
 mock
+pandas
```

Comment:
So this is in the requirements.txt but will not be pulled in the regular library?

Reply:
Yes
![image](https://user-images.githubusercontent.com/18621425/91092842-76c7f280-e60d-11ea-8dc9-431502674ddd.png)